extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -733,7 +750,7 @@ static public function setVarById( $cv_variable_id, $city_id, $value, $reason=nu
 				case "wgLanguageCode":
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this "switch/case" number of lines from 14 to at most 10, for example by extracting code into function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1151)